soc/riscv/riscv-privilege/nuclei/Kconfig.defconfig.series

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@fujitsu.com>
```

Comment:
please fixup original commit, no need to patch this later

Reply:
No longer exists this file, the file was removed when soc/riscv/riscv-privilege/nuclei moved.